### ISSUE TITLE
Update handlebars sub-dependency?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#772]
Looks like Travis is failing due to https://www.npmjs.com/advisories/755

```sh
$ npm audit fix

npm WARN postcss-html@0.23.7 requires a peer of postcss-syntax@^0.10.0 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-markdown@0.23.7 requires a peer of postcss-syntax@^0.10.0 but none is installed. You must install peer dependencies yourself.

added 2 packages from 37 contributors, removed 3 packages and updated 2 packages in 7.185s
fixed 3 of 4 vulnerabilities in 32631 scanned packages
  1 vulnerability required manual review and could not be updated


$ npm audit

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ handlebars                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.0.13                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ hbs                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ hbs > handlebars                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/755                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in 32575 scanned packages
  1 vulnerability requires manual review. See the full report for details.


$ npm ls handlebars
blurts-server@0.0.1 /Users/pdehaan/dev/github/mozilla/blurts-server
├─┬ express-handlebars@3.0.0
│ └── handlebars@4.1.0
├─┬ hbs@4.0.1
│ └── handlebars@4.0.5
└─┬ jest@23.5.0
  └─┬ jest-cli@23.5.0
    └─┬ istanbul-api@1.3.1
      └─┬ istanbul-reports@1.3.0
        └── handlebars@4.1.0  deduped
```

_Originally posted by @pdehaan in https://github.com/mozilla/blurts-server/pull/765#issuecomment-463808709_

**Comment by pdehaan** [comment]
Looks like we're using **hbs@4.0.1** (latest) which uses a _slightly_ older version of **handlebars@4.0.5** which is below the patched **>=4.0.13** version.

But it looks like **hbs@4.0.1** is the latest version:

```sh
$ npm info hbs version # 4.0.1
```

**Comment by pdehaan** [comment]
I can't figure out how to fix that nested **hbs@4.0.1** -> **handlebars@4.0.5** dependency, unless we use something like https://github.com/mozilla-services/audit-filter (which replicates the old .nsprc `exceptions` behavior).
